### PR TITLE
make systemd config check use a valid shell

### PR DESCRIPTION
**Pull request by sni**
Usually the naemon account is a system user and has no login shell. So in order
to make the config preflight check work, we have to specify a valid shell.

Signed-off-by: Sven Nierlein <sven@nierlein.de>

### CODE REVIEW

File: daemon-systemd.in [hunk 1]
```diff
@@ -10,8 +10,8 @@ PIDFile=@lockfile@
 PermissionsStartOnly=true
 ExecStartPre=-/usr/bin/mkdir /var/run/naemon
 ExecStartPre=/usr/bin/chown -R naemon:naemon /var/run/naemon/
-ExecStartPre=su - naemon -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
-ExecStart=@bindir@/naemon -d @pkgconfdir@/naemon.cfg
+ExecStartPre=su - naemon -s /bin/sh -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
```
Comment (tvestelind) [inline daemon-systemd.in]: I would like it if we used the long options `--login`, `--shell`, `--command`, `--verify-config`, `--precache-objects` for clearness purposes.

File: daemon-systemd.in [hunk 2]
```diff
@@ -10,8 +10,8 @@ PIDFile=@lockfile@
...
+ExecStart=@bindir@/naemon -ud @pkgconfdir@/naemon.cfg
```
Comment (tvestelind) [inline daemon-systemd.in]: I would like it if we used the long options `--use-precache-objects` and `--daemon` for clearness purposes.

Comment (sni) [inline daemon-systemd.in]: that might break some "check_procs" when uses filter for the naemon process, so i'd like to keep it like it is.

Comment (tvestelind) [inline daemon-systemd.in]: What do you mean? You're changing the flags anyway.

Also, this is what long options are for...

Comment (sni) [inline daemon-systemd.in]: You are right, we do change the flags already. Ok :-) You got me

**Comment by sni** [comment]
Here we go

**Pull request merged**